### DISCUSSION TITLE
Q: cURL is needed for?

### DISCUSSION THREAD

**Discussion by Djago** [RainbowMiner/RainbowMiner#1693] [Q&A] [answered]
Hi!

I've noticed that now when RBMiner starts it tells me that I don't have **cURL** (which I don't, so it's OK). Do I need to have cURL? What for? I've searched in the documentation to no avail, and only found it here and there on the release notes, but it's not clear for me if I have to install it in order to gain something.

Regards

**Comment by RainbowMiner** [comment, marked as answer]
Hi, 

I have added complete cURL integration into RainbowMiner as a substitute for the Powershell web access functions. If you run into problems, like frequent "Asyncloader crashed" or pools not updating or sudden "No miners avail", then you can switch to cURL. There is a chance such problems could go away, then. On Linux, you would have to install it (using apt install curl), on Windows 10 there is either a Windows version or I have supplied the GNU curl version in the `.\Includes` folder.

If you don't experience any problems, there is no need to enable cURL.

If you want to enable it at any time, just create an empty text file `curl.txt` in your RainbowMiner folder.

**Comment by Djago** [comment]
Thanks!
